api/v1alpha1/limitador_types.go

```diff
@@ -103,6 +109,47 @@ func (l *Limitador) Limits() []RateLimit {
 	return l.Spec.Limits
 }
 
+func (l *Limitador) GetImage() string {
+	image := fmt.Sprintf("%s:%s", l.GetImageRepository(), l.GetImageTag())
+	return helpers.FetchEnv(ImageEnvVarKey, image)
+}
```

Comment:
There's concern / discussion on whether we still want to expose the env var or should the env var take precedence or not and also should this be exposed at all via the CR spec as different limitador versions has different limit file formats

Reply:
Cool, I'll refactor this 👍